keyboards/kbdfans/kbd75/keymaps/tucznak/config.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#ifdef MANUFACTURER
```

Comment:
As above:
you could remove the ifdef's for these. They should always be defined.

It's not necessary, but would make it a bit cleaner looking.